src/corva/models/base.py

```diff
@@ -17,3 +24,7 @@ class RawBaseEvent(abc.ABC):
     @abc.abstractmethod
     def from_raw_event(event: Any) -> Sequence[RawBaseEvent]:
         pass
+
+    @classmethod
+    def get_app_type(cls) -> AppType:
```

Comment:
issue(blocking): Event payload types and type defined in manifest.json are not the same. What do you think about removing this method from the interface and do types comparison in "validate_app_type_context" method.